portfolio/src/main/webapp/about.js

```diff
@@ -0,0 +1,79 @@
+let answer = -1;
```

Comment:
Before I review this whole file again, I would deeply appreciate either:

Method comments: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

Or a file comment: https://google.github.io/styleguide/jsguide.html#jsdoc-top-file-level-comments

Which explain what this code does. It will make it much easier for us to review the logic that way :)

Reply:
Added documentation